torch/_dynamo/variables/builder.py

```diff
@@ -2385,6 +2385,9 @@ def handle_traced_output(example_value, tx, proxy, options, subclass_type, targe
     ):
         set_example_value(proxy.node, example_value)
         return ConstantVariable.create(example_value, **options)
+    elif isinstance(example_value, int) and (proxy.node.target in ["__round__"]):
+        set_example_value(proxy.node, example_value)
+        return ConstantVariable.create(example_value, **options)
```

Comment:
Merge this with below? If you can unconditionally let this through that would be good too lol